tough/src/target_name.rs

```diff
@@ -258,15 +258,15 @@ fn resolved_8() {
 
 #[test]
 fn uncleaned_1() {
-    let name = r#"~/\.\."#;
+    let name = r"~/\.\.";
```

Comment:
These are technically fine. The spec calls for "zero or more #" to demarc raw strings, so the existing code is fine (and probably what is expected in most cases). This was something `cargo clippy --fix` changes though, so I decided to include it to make any future uses of `--fix` have less noise. But it really isn't necessary, so if folks would rather see this kept as-is, I can drop this part of the patch.

Reply:
I'm ok with this, though I admit I always add the `#` charactrers to raw strings out of habit.